main/main.ts

```diff
@@ -135,6 +135,13 @@ app.on('open-url', (e: Event, u: string) => {
     e.preventDefault();
     shell.openExternal(u);
 });
+// we use once here because only the first open-file event might be missed by nyaovim-app
+app.once('open-file', (e: Event, p: string) => {
+    // open-file event might be sent before ready event is emitted
+    // put it in argv to let nyaovim-app to pick it up later
+    process.argv.push(p);
+    e.preventDefault();
+});
```

Comment:
Nice catch!  But it looks not related to this PR directly.  Could you create another PR for this to manage history of development better?